.github/workflows/graviton-build-test-push-workflow.yml

```diff
@@ -0,0 +1,308 @@
+name: Graviton Smoke Test WorkFlow
+on:
+  push:
+    branches:
+      - CSPL_2920
+      - CSPL_xx
```

Comment:
Before merging, should these be changed to develop?

Reply:
Yes, will be cleaning this up.